src/osd/PrimaryLogPG.cc

```diff
@@ -4937,6 +4937,8 @@ int PrimaryLogPG::do_osd_ops(OpContext *ctx, vector<OSDOp>& ops)
   ObjectState& obs = ctx->new_obs;
   object_info_t& oi = obs.oi;
   const hobject_t& soid = oi.soid;
+  bool skip_data_digest = osd->store->get_type() == "bluestore" &&
```

Comment:
probably this should be a property of the underlying store. i.e. we can just hard wire it to true for bluestore and false for filestore.